melosys-eessi-app/src/test/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/lovvalg/A010MapperTest.java

```diff
@@ -66,7 +68,24 @@ public void mapTilSed_medTilleggsbestemmelseBestemmelseIkkeGyld_tilleggsBestemme
         assertThat(medlemskap.getVedtak().getGjelderperiode().getStartdato()).isNotNull();
         assertThat(medlemskap.getVedtak().getGjelderperiode().getSluttdato()).isNotNull();
     }
+    @Test
```

Comment:
Du kan forresten gjerne skrive om fra junit4 - junit5 i test-klassene du toucher innom. 

Innebærer for det meste bare:
```
@Before -> @BeforeEach
@org.junit.Test -> @org.junit.jupiter.api.Test
@RunWith(MockitoRunner.class) -> @ExtendWith(MockitoExtension.class)
```

Grunnen til det er at junit4 ikke er inkludert i nye spring-versjoner, og vi har lagt til en liten "hack" i melosys-eessi for at både junit4 og junit5-tester skal kjøres